src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -322,6 +285,20 @@ const AppWrapper = (props: { children: ReactNode }) => {
         throw e;
       }
     }
+    // @TODO - foconnor: This is a temp hack for development to be removed pre-release.
+    // These items are removed from the secure storage on re-install to re-test the on-boarding for iOS devices.
+    const appAlreadyInit = await Agent.agent.basicStorage.findById(
+      MiscRecordId.APP_ALREADY_INIT
+    );
+    if (!appAlreadyInit) {
```

Comment:
This check should be done before launching the Signify client because the bran gets deleted in the block.

I guess, the database won't be started without `Agent.agent.start` - that was actually why I listed DTIS-921 ahead of DTIS-920 (this PR) so that the start method would already be broken up. :P 

I think we can take the risk - but DTIS-921 needs to fix the order of this and be done before we create the next TestFlight deployment.

Reply:
So idea overall would be:
- Load the database layer immediately and populate Redux
- Then either we are still in onboarding and have no agent started
- Or we immediately call start to connect to KERIA and load the callback handlers for notifications

Reply:
@iFergal Shall we fix it on this PR?

Reply:
@bao-sotatek No it's OK, otherwise this PR would get too big and do too much of DTIS-921. It's OK like this for now, since it will change soon